docs/references/api/deviceshifu-configmap.md

```diff
@@ -35,13 +35,50 @@ DeviceShifuInstructions are all the commands and settings that ***deviceShifu***
 
 - **instructionSettings** ([DeviceShifuInstructionSettings](#deviceshifuinstructionsettings))
 - **instructions** (map[string]DeviceShifuInstruction)
+**example**
+```yml
+instructions: |
+  instructions:
+    get_value: # The name of the command you want to set
+      protocolPropertyList:      # The parameters of the command have different configurations according to different protocols
+```
     - **[DeviceShifuInstruction](#deviceshifuinstruction)**
 
 ## DeviceShifuInstruction
 
 DeviceShifuInstruction is the command that ***deviceShifu*** can receive.
 
--  **protocolPropertyList** (map[string]string)<br/>parameters of the ***deviceShifu*** command, which have various configurations according to different protocols. Please refer to the  examples for more [examples](https://github.com/Edgenesis/shifu/tree/main/examples).
+-  **protocolPropertyList** (map[string]string)([DeviceShifuprotocolPropertyList](#deviceshifuprotocolpropertylist))<br/>parameters of the ***deviceShifu*** command, which have various configurations according to different protocols. Please refer to the  examples for more [examples](https://github.com/Edgenesis/shifu/tree/main/examples).
+
+## DeviceShifuprotocolPropertyList
+
+DeviceShifuprotocolPropertyList   parameters of the ***deviceShifu*** command, which have various configurations according to different protocols.The following shows the use of MQTT protocol and OPC UA protocol. Please refer to the  examples for more [examples](https://github.com/Edgenesis/shifu/tree/main/examples).
```

Comment:
remove extra spaces after "DeviceShifuprotocolPropertyList"
add a space after "different protocols."
remove extra spaces between "the  examples"